Resources/public/js/views/plugins/cof-createcontent-universaldiscoveryplugin.js

```diff
@@ -77,7 +78,25 @@ YUI.add('cof-createcontent-universaldiscoveryplugin', function (Y) {
             } else {
                 tabCreateLabel.removeClass(CLASS_HIDDEN);
             }
-        }
+        },
+
+        /**
+         * Close the discovery widget after publishing content.
+         *
+         * @protected
+         * @method _closeDiscoveryWidget
+         * @param event {Object} event facade
+         */
+        _closeDiscoveryWidget: function (event) {
+            var host = this.get('host'),
+                selectedContent = host.get('multiple') ? [event] : event;
+
+            host._set('selection', selectedContent);
```

Comment:
I don't know at the moment. As said before, since I'm arriving in the middle of the feature, I'm completely lost, I need first to understand "where we are" and then "where we are going" with that...